build_and_run.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+ExportEnvironmentVars()
+{
+  export FLASK_APP=pydash.py
+  export FLASK_ENV=development
+  export MAIL_USERNAME=noreply.pydashtestmail@gmail.com
+  export MAIL_PASSWORD=verysecurepydashpassword
```

Comment:
Don't put the password in the run-file

Reply:
This is temporary, the deployed version will use secret credentials.

Reply:
(also see #421)